app/assets/javascripts/admin/addon/components/emoji-uploader.hbs

```diff
@@ -1,33 +1,39 @@
-<div class="emoji-uploader form-horizontal">
-  <div class="control-group">
-    <span class="label">
+<div class="form-kit">
+  <div class="form-kit__container form-kit__field form-kit__field-input-text">
```

Comment:
I like that FormKit is flexible enough to make it so we can "fake" the look for other forms for cases like this :)